CommandLine/testing/SimpleTests/math_test.sog/create.edl

```diff
@@ -167,7 +167,14 @@ gtest_assert_eq(point_distance_3d(1.0, 2, 3.0, 3, 5.0, 9), 7);
 gtest_assert_eq(point_direction(0, -1, 2, -3), 45);
 gtest_assert_eq(point_direction(3.0, 2, 1, 0.0), 135);
 
-
+// point_in_rectangle test, values expected from GMS v1.4.1804
+gtest_assert_eq(point_in_rectangle(0, 0, 1, 1, 3, 3), false);
+gtest_assert_eq(point_in_rectangle(0, 1, 1, 1, 3, 3), false);
+gtest_assert_eq(point_in_rectangle(1, 0, 1, 1, 3, 3), false);
+gtest_assert_eq(point_in_rectangle(1, 1, 1, 1, 3, 3), true);
+gtest_assert_eq(point_in_rectangle(2, 2, 1, 1, 3, 3), true);
+gtest_assert_eq(point_in_rectangle(3, 3, 1, 1, 3, 3), true);
+gtest_assert_eq(point_in_rectangle(4, 4, 1, 1, 3, 3), false);
```

Comment:
@JoshDreamland I wrote this test to fail the first time.

Why didn't it fail, should I fix that, and how do I fix that?
https://travis-ci.org/enigma-dev/enigma-dev/jobs/378662566#L2469

Reply:
Two questions:
1. What if you just do `gtest_assert_eq(true, false);`
2. Why are you not using `gtest_assert_true(...)`?

Reply:
1. I don't know, setting up gtest for me is non-trivial, so you should test locally, but also I think you should test the converse: `gtest_assert_eq(false, true);`
2. I didn't realize it existed until after I wrote the test, but then figured it wouldn't matter anyway. Also, I kind of wanted clarification as to what the difference there really is, i.e. when to use one and not the other.

Reply:
Ok, let's see what #1234 does.

Reply:
See my newer comment, it turns out this had to do with not pulling branch master first for some reason (I don't know, Travis/Git bug maybe?).